exercises/chapter12/test/Main.purs

```diff
@@ -67,16 +78,21 @@ main =
     suite "countCharacters" do
       test "exists" do
         chars <- countCharacters $ Path.concat [ inDir, "foo.txt" ]
```

Comment:
What about just changing `foo.txt` so there are no line breaks?

Reply:
Yes, it will do the trick too